dasharo/README.md

```diff
@@ -83,7 +83,7 @@ by adding it to 3mdeb-secpack repository.
 
 * Decrypt new key
 
-```
+```shell
 $ gpg --output <new-release-key-name>-priv.asc --decrypt <new-release-key-name>-priv.asc.gpg
```

Comment:
@macpijan somehow despite enforcing that rule MD014, there is still stuff like here starting with `$`

Reply:
@macpijan ping?

Reply:
@pietrushnic Correct. As I said here: https://github.com/3mdeb/3mdeb-secpack/pull/102#discussion_r1358077658 these must be fixed manually. If there is also output in a single block, the rule does not apply and the dollar is accepted. We would need to manually rework all of the blocks to have output in a separate block from the command.

Reply:
@macpijan ok, your comment was not clear for me what `majority` covers. Resolving.

Reply:
Issue added to address that problem: https://github.com/3mdeb/3mdeb-secpack/issues/110